### PR TITLE
[W5.8][W14-A3]Wang Jinyi

### PR DESCRIPTION
**Pull request by WJY-norainu**
added 3 NFRs

### CODE REVIEW

**Review by kowshik-sundararajan — COMMENTED** [review]
Your understanding of NFR is quite good; do look through the comments before closing the PR.

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should respond within 5 seconds.
```
Comment (kowshik-sundararajan) [inline doc/DeveloperGuide.md:97]: Could you be more specific for this NFR?

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -94,6 +94,9 @@ Use case ends.
...
+6. Should be backward compatible with data produced by earlier versions of the AddressBook.
+7. Should be usable by people with no knowledge about command line input.
```
Comment (kowshik-sundararajan) [inline doc/DeveloperGuide.md:99]: Good NFR, this is quite specific!

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (WJY-norainu) [reply doc/DeveloperGuide.md:97]: Thank you for reviewing!
Do you mean I should include respond to which command as well?

Comment (kowshik-sundararajan) [reply doc/DeveloperGuide.md:97]: More like `Should not take more than 5 seconds to respond to a command entered by the user` or similar to that.

Although it is just a suggestion, if you feel like it is specific enough, then don't worry about it.

**Pull request closed without merging**